src/main/pages/che-7/administration-guide/con_a-workspace-example.adoc

```diff
@@ -32,50 +32,48 @@ components:
 This table provides the memory requirements for each workspace component:
 
 [cols="1,3,1,1", options="header"]
-.Total Workspace Memory Requirement and Limit
+.Total workspace memory requirement and limit
 |===
 |Pod
-|Container Name
-|Default Memory Limit
-|Default Memory Request
+|Container name
+|Default memory limit
+|Default memory request
 
 |Workspace
 |theia-ide (default `cheEditor`)
-|512Mi
-|512Mi
+|512 Mi
```

Comment:
This is probably my ignorance but what are these units exactly? I assume they're mebibyes but `mebi` units are abbreviated as `MiB`, `GiB` and so on, as far as I know.

Reply:
Good point. I didn't realize this. Will fix.

Reply:
Pushed a fix (converted to MiB etc.).